<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emailMsgForm</class>
 <widget class="QWidget" name="emailMsgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_up">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_down">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>30</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>你在哪</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_no">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_yes">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);</string>
   </property>
   <property name="text">
    <string>√</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_opt">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lb_opt</zorder>
  <zorder>pb_icon</zorder>
  <zorder>lb_up</zorder>
  <zorder>lb_down</zorder>
  <zorder>pb_no</zorder>
  <zorder>pb_yes</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
